wrapping/python/plugins/debugging_helper.py

```diff
@@ -0,0 +1,99 @@
+"""
+This code can be used to debug your python based DREAM3D-NX filter. There are a
+number of bits of code that you will need to change in order for you to be able
+to debug.
+
+
+"""
+from typing import List
+import simplnx as nx
+
+# ------------------------------------------------------------------------------
+# This NEEDS to be executed here so that we can load the python based plugin
+#
+# You will need to REPLACE the name of the plugin and the name of the filter
+# that you are trying to debug in the next 3 lines of code
+# ------------------------------------------------------------------------------
+import DataAnalysisToolkit
+nx.load_python_plugin(DataAnalysisToolkit)
```

Comment:
This should only be necessary if you are trying to load a pipeline with a python filter from this plugin. For just programmatically running a filter in python you could omit this though having shouldn't harm anything.

Reply:
I thought we decided that it was needed?

Reply:
It's needed for loading a pipeline from json for sure.